apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/http/get/GetAllExchanges.java

```diff
@@ -50,7 +50,7 @@ public JSONStreamAware processRequest(HttpServletRequest req) throws AplExceptio
 
         JSONObject response = new JSONObject();
         JSONArray exchanges = new JSONArray();
-        try (DbIterator<Exchange> exchangeIterator = Exchange.getAllExchanges(firstIndex, lastIndex)) {
+        try (DbIterator<Exchange> exchangeIterator = exchangeService.getAllExchanges(firstIndex, lastIndex)) {
```

Comment:
I think, that better to replace it by proper filter applied on service/repo level (also it will help to not expose DbIterator)

Reply:
I think the changing 'old rest api' is out of the scope in current task for 'refactoring old static' classes into 'three layer components'.